src/android/test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * test module
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { View } from './view';
+
+export default class Test {
+
+	constructor (app, module) {
+		this.app = app;
+		this.module = module;
+	}
+
+	startup () {
+		this.app.logger.info(this.module.id + ' startup');
+	}
+
+	shutdown () {
+		this.app.logger.info(this.module.id + ' shutdown');
+	}
+
+	static createView (opts = {}) {
```

Comment:
yep, agreed with that. i'm fine with just exporting the classes directly and requiring them instead of using the factory pattern. but we just need to make sure we make that clear that we're moving away from idiomatic titanium to something that's mixed (factory for titanium legacy, class constructor for hyperloop + titanium.next).